### PR TITLE
DATA-37 - Report format changes.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR changes some things to the reports such as:

- Remove user role column for the three reports.
- Add section name column to last page accessed report.
- Update README.md

## Description:

- [ ] @andrey-canon

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ GOOGLE_OAUTH_CREDENTIALS came from downloaded Google oAuth credentials in json f
 
 ## Running
 
-python3 ./fetch_report.py --report "supported-report-name" --config-file "path-to-config-file"
+python3 ./fetch_report.py --report "supported-report-name" --config-file "path-to-config-file" --oauth-config-file "path-to-oauth-credentials-file"
```
Comment (andrey-canon) [inline README.md]: Does this change have any relationship with this PR ?

Comment (Squirrel18) [reply README.md]: No, but it is necessary to update it.

Comment (andrey-canon) [reply README.md]: could you open a new PR to do this ?

Comment (Squirrel18) [reply README.md]: What is the problem with?

Comment (andrey-canon) [reply README.md]: it's not easy track a change that is hiding in a PR without relation.

Comment (Squirrel18) [reply README.md]: It is an update to the README, and you can always use a blame in the git history, anywise, it is done.

**Review by andrey-canon — APPROVED** [review]

**Pull request merged**